src/Apps/IdentityVerification/Processing.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Button, Sans, Serif } from "@artsy/palette"
+import { AppContainer } from "Apps/Components/AppContainer"
+import React from "react"
+
+import { Meta, Title as HeadTitle } from "react-head"
+
+export const Processing = () => {
+  return (
+    <AppContainer>
+      <HeadTitle>Artsy | ID Verification</HeadTitle>
+
+      <Meta
+        name="viewport"
+        content="width=device-width, initial-scale=1, maximum-scale=5 viewport-fit=cover"
```

Comment:
Is this line necesary?

Reply:
I thought it was, but it looks OK without it.